create_grids/incident/sps/install_maraston11_pickles.py

```diff
@@ -93,7 +91,7 @@ def make_grid(model, imf, extension, input_dir):
             print(iZ, ia, fn)
             ages_, _, lam_, flam_ = np.loadtxt(fn).T
 
-            flam = flam_[ages_ == age_Gyr]
+            flam = flam_[ages_ == age_Gyr] * nJy
```

Comment:
are you sure that flam is in these units here? `nJy` are a unit of spectral flux density in terms of frequency

Reply:
Not sure what happened here - correcting this to erg / s / Angstrom / cm**2